### PR TITLE
Pacemaker is unsupported for 3.2

### PR DESCRIPTION
**Pull request by brenton**

### CODE REVIEW

File: playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml [hunk 1]
```diff
@@ -35,6 +35,12 @@
 
   - fail:
       msg: >
+        This upgrade does not support Pacemaker:
+        https://docs.openshift.com/enterprise/3.1/install_config/upgrading/pacemaker_to_native_ha.html
```
Comment (brenton) [inline playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml]: I'll make this generic enough to handle Origin links too.

File: playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml [hunk 2]
```diff
@@ -33,6 +33,15 @@
         deployment types
     when: deployment_type not in ['atomic-enterprise', 'origin','openshift-enterprise', 'online']
 
+  - set_fact:
+      g_pacemaker_upgrade_url_segment: "{{ 'org/latest' if deployment_type =='origin' else '.com/enterprise/3.1' }}"
```
Comment (detiber) [inline playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml]: This can be put in vars instead of using set_fact

Comment (brenton) [inline playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml]: k

**Comment by detiber** [comment]
outside of the nit I picked about using vars instead of set_fact :+1: from me. I'm happy to see pacemaker go away :grinning:

**Comment by brenton** [comment]
aos-ci-test

**Pull request merged**